Gemfile

```diff
@@ -1,14 +1,19 @@
-# frozen_string_literal: true
```

Comment:
Please change this file to match the one we ended up merging in the usrse23 repository.

Reply:
Done in commit  c9592299ee0e5a4a74fd766731f79282a31e65aa